test/nock/post.orders.js

```diff
@@ -34,9 +36,14 @@ module.exports = nock(config.api)
             order.order_configuration.limit_limit_gtc.base_size,
             order.price
           )
-        : order.order_configuration.market_market_ioc.quote_size ||
-            order.order_configuration.market_market_ioc.base_size
+        : isBuy
+        ? order.order_configuration.market_market_ioc.quote_size
+        : multiply(
+            order.order_configuration.market_market_ioc.base_size,
+            order.price
+          )
```

Comment:
we should cleanup these test scripts. They were kind of haphazard to start with, but works for now